src/main/java/io/github/antishake/AntiShake.java

```diff
@@ -257,27 +259,32 @@ void convolve(ArrayList<Double> impulseResponseSamples, ArrayList<Coordinate> ac
 
   /**
    * Returns the value of the given key from the config.properties file
+   * TODO Write a util class that returns value by type and default value
    *
    * @param key
    * @return value
    */
-  private static String getPropertyValue(String key) {
-    return properties.getProperty(key);
+  private static String getPropertyValue(String key, String defaultValue) {
+    if (properties.containsKey(key)) {
+      return properties.getProperty(key);
+    } else {
+      return defaultValue;
+    }
   }
 
   /**
    * Loads all the properties from config.properties file and assigns to
    * appropriate static variables
    */
   private static void loadProperties() {
-    SPRING_CONSTANT = Double.parseDouble(getPropertyValue("SPRING_CONSTANT"));
-    CIRCULAR_BUFFER_IN_SEC = Double.parseDouble(getPropertyValue("CIRCULAR_BUFFER_IN_SEC"));
-    SAMPLING_RATE_IN_HZ = Double.parseDouble(getPropertyValue("SAMPLING_RATE_IN_HZ"));
+    SPRING_CONSTANT = Double.parseDouble(getPropertyValue("SPRING_CONSTANT", "10"));
```

Comment:
Rather than hardcoding here, we can assign values to the final Strings that we have created to assign the values from Properties file, and remove else in line 270.